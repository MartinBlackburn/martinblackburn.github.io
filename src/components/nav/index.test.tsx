// libraries
import { render } from "@testing-library/react";

// component under test
import Nav from "./";

describe("Nav component", () => {
    test("renders correctly", () => {
        const { asFragment } = render(<Nav />);

        expect(asFragment()).toMatchSnapshot();
    });
});
