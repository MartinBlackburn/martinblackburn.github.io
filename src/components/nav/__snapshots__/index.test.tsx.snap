// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders correctly 1`] = `
<DocumentFragment>
  <nav
    class="nav"
  >
    <div
      class="nav__icon"
    >
      <svg
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M3,6H21V8H3V6M3,11H21V13H3V11M3,16H21V18H3V16Z"
        />
      </svg>
    </div>
    <ul
      class="nav__links"
    >
      <li>
        <button
          class="nav__button"
        >
          Home
        </button>
      </li>
      <li>
        <button
          class="nav__button"
        >
          Résumé
        </button>
      </li>
      <li>
        <div
          class="nav__title"
        >
          Projects
        </div>
        <ul>
          <li>
            <button
              class="nav__button"
            >
              Shuffleboard
            </button>
          </li>
        </ul>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
